web/flat-web/src/pages/JoinPage/index.tsx

```diff
@@ -22,6 +22,8 @@ export const JoinPage = observer(function JoinPage() {
     const [isLogin, setIsLogin] = useState(false);
     const { width } = useWindowSize(1080);
 
+    const iframeRef = useRef<HTMLIFrameElement | null>(null);
```

Comment:
```suggestion
    const iframeRef = useRef<HTMLIFrameElement>(null);
```

`| null` is unnecessary. See https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts#L1053

Reply:
I will immediately open another PR fix